src/main/resources/application.yml

```diff
@@ -70,3 +70,5 @@ application:
   kafka:
     topic-pattern: ${ENV:folio}.(.*\.)?data-export.job.update
     group-id: ${ENV:folio}-mod-bulk-operations-events-group
+  file-uploading:
+    max-retry-count: 50
```

Comment:
Let's try to play with it and start from 25. Make it as configurable from env vars - move it also in the module descriptor ends section to have possibility to configure as env var.